tools/tools.go

```diff
@@ -24,11 +24,14 @@ package tools
 
 import (
 	_ "github.com/elastic/go-licenser"
+	_ "github.com/ettle/strcase"
```

Comment:
Why are imports in this file added?

Reply:
Moved dependencies to a separate `go.mod/sum` 👍